src/midi/midi_fluidsynth.cpp

```diff
@@ -359,15 +369,26 @@ bool MidiHandlerFluidsynth::Open([[maybe_unused]] const char *conf)
 	// apply custom reverb settings if provided
 	if (reverb_enabled && reverb.size() > 1) {
 		if (reverb.size() == 4) {
-			apply_setting("reverb room-size",
-			              reverb[0],
-			              reverb_room_size,
-			              0.0,
-			              1.0);
-
-			apply_setting("reverb damping", reverb[1], reverb_damping, 0.0, 1.0);
-			apply_setting("reverb width", reverb[2], reverb_width, 0.0, 100.0);
-			apply_setting("reverb level", reverb[3], reverb_level, 0.0, 1.0);
+			reverb_room_size = apply_setting("reverb room-size",
+			                                 reverb[0],
+			                                 reverb_room_size,
+			                                 0.0,
+			                                 1.0);
+			reverb_damping = apply_setting("reverb damping",
+			                               reverb[1],
+			                               reverb_damping,
+			                               0.0,
+			                               1.0);
+			reverb_width = apply_setting("reverb width",
+			                             reverb[2],
+			                             reverb_width,
+			                             0.0,
+			                             100.0);
+			reverb_level = apply_setting("reverb level",
+			                             reverb[3],
+			                             reverb_level,
+			                             0.0,
+			                             1.0);
```

Comment:
Good catch! I think some clarification is needed though as this is a bit confusing. Clearly, `apply_setting` doesn't apply anything; it returns the passed in value if it's within the valid range, or the passed in default otherwise.

So I recommend to rename it to `validate_setting` and perhaps even include a short comment above the function to explain what it's doing.